ccm_web/server/src/api/api.service.ts

```diff
@@ -21,4 +29,44 @@ export class APIService {
       }
     }
   }
+
+  async getCourseName (userLoginId: string, courseId: number): Promise<string | null> {
```

Comment:
What is the reason for having an API call for fetching the course name?

Reply:
We need to know the course name so it can be displayed in the UI. Pulling the course name from the launch variables is not ideal because once it's changed, we'd need to keep it updated separately in the session. I suppose we could do that, but this seemed okay as well.

Reply:
ok